README.md

```diff
@@ -1,19 +1,18 @@
-# Status bindings for go-ethereum
+[![Unit Tests](https://ci.status.im/buildStatus/icon?subject=Unit%20Tests&job=status-go%2Ftests-develop)](https://ci.status.im/job/status-go/job/tests-develop/) [![Unit Tests](https://ci.status.im/buildStatus/icon?subject=Functional%20Tests&job=status-go%2Ftests-rpc-develop)](https://ci.status.im/job/status-go/job/tests-rpc-develop/) [![codecov](https://codecov.io/github/status-im/status-go/graph/badge.svg?token=mQIkVPUdVv)](https://codecov.io/github/status-im/status-go)
 
-[![GoDoc](https://godoc.org/github.com/status-im/status-go?status.svg)](https://godoc.org/github.com/status-im/status-go)
-[![Build Status](https://travis-ci.org/status-im/status-go.svg?branch=develop)](https://travis-ci.org/status-im/status-go)
+# status-go
+
+`status-go` is the backbone library of [Status Desktop](github.com/status-im/status-desktop) and [Status Mobile](https://github.com/status-im/status-mobile). It acts as a "backend" for both applications and implements most of the Status apps business logic. 
+
+A comprehensive list of `status-go` functionality can be found in [Brief overview](https://www.notion.so/WIP-Brief-overview-1578f96fb65c8042a166ecd79a6b1cb6?pvs=4).
```

Comment:
Is that notion page open to the public? Not sure if this is temporary, sounds like content that should be hosted in the repo.